Project-JS/clear_localstorage.js

```diff
@@ -0,0 +1,14 @@
+/*Exposing a window-level function to clear localStorage values who's key includes the substring 'optimizely' 
+*/
+window.clearOptlyLocalStorage = function(){
+    for (var key in window.localStorage){
+        if(key.indexOf('optimizely') !== -1){
+            window.localStorage.removeItem(key)
+        }
+    }
+}
+//The function can be called from Optimizely Project JS based on an available boolean flag, such as a cookie's value 
+var should_clear = window.customer_storage_flag_boolean;
+if(should_clear){
+    clearOptimizelyLocalStorage();
```

Comment:
Looks like this should be named `clearOptlyLocalStorage` if you are trying to use the function above.